### PR TITLE
Decreases Captain Bank account to 300k, fixes moebius name in paycheck

### PR DESCRIPTION
**Pull request by ValoTheValo**
## About The Pull Request
lowers the captains bank account from 2 million to 300k to make the captain be more mindful about spending large amounts of money or just donating it and breaking the game economy, also fixes a minor misnomer

## Why It's Good For The Game
this makes the captain no longer able to break the game economy at will by giving hundreds of thousands of credits away without consequence

## Testing
testing
testing2

## Changelog
:cl:
tweak: changed name of Moebius on paychecks from Corp to Labs
balance: Captain now only has 300k in thier department account
/:cl:

### CODE REVIEW

**Comment by DiaFRAME444** [comment]
Huge improvement but still way too much. What's the rationale?

**Comment by ValoTheValo** [comment]
> Huge improvement but still way too much. What's the rationale?

it should last like 3 hours if the paycheck is 50k for half an hour, i don't have data on how much the average half hourly paycheck is so i made a guess from what i remember on highpop, it can be further adjusted if necessary

**Comment by Mycah142** [comment]
If every department listed as dependents were fully staffed, they'd be **10,050 every half hour.** 300k is 29.850746268656716417910447761194 total payments. Or 14.925373134328358208955223880597 hours.

TL;DR, 300k = 15 hours for fully staffed departments for every single one of those hours.

Here's the breakdown I just did:

FO: 600

DEPARTMENTS (auto 500 without people)

IH: 5600 max
Commander: 600
Sergeant: 450
Inspector: 450, max of 2. 900 max
Specialist: 450, max of 2. 900 max
Operative: 375, max of 6. 2250 max

Techno: 3350 max
Exaltant: 600
Technomancer: 450, max of 5. 2250 at max

Civ
500, workers don't make normal wages.

File: code/game/jobs/department.dm [hunk 1]
```diff
@@ -44,7 +44,7 @@
 	In future, we will implement largescale missions and research contracts to earn money, and then set it
 	to a much lower starting value
 	*/
-	account_initial_balance = 2000000
+	account_initial_balance = 300000
```
Comment (Firefox13) [inline code/game/jobs/department.dm:47]:
```suggestion
	account_initial_balance = 600000
```
...how about we still give captains some leeway?

Comment (Mycah142) [reply code/game/jobs/department.dm:47]: To what? There's 15 hours of payments to fully staffed dependants within 300k, and the likelihood of techno/IH both being fully staffed for 15 hours is literally impossible.

Captain using the money for frivolous shit is still doable. 600k is enough to both break the trade beacons, AND pay fully staffed dependants for 20 hours.

300k means breaking the trade beacons isn't as brute force, and more careful purchases while still being able to fund dependants for 6-10 hours.

Comment (Firefox13) [reply code/game/jobs/department.dm:47]: Just so the captain still has some *leeway*. They are a rich person. If they can't flex they're free 1.7 millions, let them flex atleast 300k. If they misuse it we'll have to talk to them instead.

Money is one of the more distinct captain's powers.

Comment (TheLeopold) [reply code/game/jobs/department.dm:47]: According to Mycah's calculations Captain needs around 120k for 6h round. That means Captain is free to flex their 180k.
I don't feel like 480k flex-money is right.

**Comment by Kurgis** [comment]
> Whoever decided 2mil in the first place was a numerical dumbass.

Initially when added, plans were to have the Overmap (which would be the main point of gameplay, the ship exploring stuff) have important expenditures a captain would have to decide to make. When the ~2mil was initially added, it was under the knowledge that it was way too much as is, but that it would be resolved later as the overmap develops.

I'm in support of giving the captain less "lee-way", especially as someone who likes playing the role. It can trivialize decisions, some character interactions, and of course the ship's economy most obviously. I think it'd be nice to implement some - _shock therapy_ - to the Eris, which has long been inundated with the "Command Cash". I think it'd would even _help_ to start it out too low, and raise it again as needed through the process of observing round play with the changes.

**Comment by Mycah142** [comment]
> Initially when added, plans were to have the Overmap (which would be the main point of gameplay, the ship exploring stuff) have important expenditures a captain would have to decide to make. When the ~2mil was initially added, it was under the knowledge that it was way too much as is, but that it would be resolved later as the overmap develops.

This has been so forgotten that other possible plans for the command account cash have been discussed.

**Comment by SaltyOldBull** [comment]
rich people don't go around with that much liquidity in their bank accounts, it's all tied up to assets
triply so when it comes to someone so tied up in debts, the CEV Eris is the main asset of the captain, it's not liquid
as to why it's not that liquid, tax avoidance
in a gameplay perspective, this leads to the equivalent of money printing should the captain elect to make generous donations, devaluing credits even further. You can't begin to fix the Eris economy without tackling the biggest sources of magic money.

File: code/game/jobs/department.dm (hunk 1 above)
Comment (ValoTheValo) [reply code/game/jobs/department.dm:47]: yeah 480k spare credits is waaaaaaaaaaaaay too much leeway, you can still absolutely wreck the game's economy with that and brute force both guild and moebius trees

**Comment by githubuser4141** [comment]
> rich people don't go around with that much liquidity in their bank accounts, it's all tied up to assets triply so when it comes to someone so tied up in debts, the CEV Eris is the main asset of the captain, it's not liquid as to why it's not that liquid, tax avoidance in a gameplay perspective, this leads to the equivalent of money printing should the captain elect to make generous donations, devaluing credits even further. You can't begin to fix the Eris economy without tackling the biggest sources of magic money.

200 k isn't that much money in terms of people rich enough to purchase a giant yacht, unfortunately because we're LRP the "assets" that exist as in game items have no inherent value or don't exist. 30 k is really small imho, for e.g. a shipment of guns is about 5 - 7k, and that shipment is only 3 guns without ammo. 30k has essentially no wiggle room for large grants a captain might give to a department and isn't even enough for a large shipment of materials.

**Comment by Mycah142** [comment]
> 200 k isn't that much money in terms of people rich enough to purchase a giant yacht, unfortunately because we're LRP the "assets" that exist as in game items have no inherent value or don't exist. 30 k is really small imho, for e.g. a shipment of guns is about 5 - 7k, and that shipment is only 3 guns without ammo. 30k has essentially no wiggle room for large grants a captain might give to a department and isn't even enough for a large shipment of materials.

you lugnut it's 300k being proposed.

File: code/game/jobs/department.dm (hunk 1 above)
Comment (Firefox13) [reply code/game/jobs/department.dm:47]: fair

**Review by Firefox13 — APPROVED** [review]
Без имени-1
